src/utils/csv.ts

```diff
@@ -0,0 +1,9 @@
+import * as fs from "fs";
+import neatCsv from "neat-csv";
+
+export const getCSVRows = async (filePath: string) => {
+  let results: Array<Object> = [];
```

Comment:
We use the `[]` suffix rather than the `Array<>` version of it:

```suggestion
  let results: Object[] = [];
```

Reply:
Also, is `Object` the best type to use here? Is there nothing more specific we can use?

Reply:
fix in 71fbe64d1f3591d78979411e27e72e5165340a4b